blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -20,70 +36,123 @@ const SmallPromoItem = ({ customFields, arcSite }) => {
 			// does not need website section nor label because no overline
 			// does not need byline because no byline shown
 			filter: `{
-      _id
-      description {
-        basic
-      }
-      display_date
-      type
-      headlines {
-        basic
-      }
-      promo_items {
-        type
-        url
-        lead_art {
-          type
-          promo_items {
-            basic {
-              type
-              url
-              resized_params {
-                800x600
-                800x533
-                800x450
-                600x450
-                600x400
-                600x338
-              }
-            }
-          }
-        }
-        basic {
-          type
-          url
-          resized_params {
-            800x600
-            800x533
-            800x450
-            600x450
-            600x400
-            600x338
-          }
-        }
-      }
-      websites {
-        ${arcSite} {
-          website_url
-        }
-      }
-    }`,
+		_id
+		description {
+			basic
+		}
+		display_date
+		type
+		headlines {
+			basic
+		}
+		promo_items {
+			type
+			url
+			lead_art {
+				type
+				promo_items {
+					basic {
+						type
+						url
+						resized_params {
+							800x600
+							800x533
+							800x450
+							600x450
+							600x400
+							600x338
+						}
+					}
+				}
+			}
+			basic {
+				type
+				url
+				resized_params {
+					800x600
+					800x533
+					800x450
+					600x450
+					600x400
+					600x338
+				}
+			}
+		}
+		websites {
+			${arcSite} {
+				website_url
+			}
+		}
+	}`,
 		}) || null;
 
-	return <SmallPromoPresentation content={content} {...customFields} />;
-};
+	if (!customFields?.itemContentConfig) return null;
 
-const SmallPromo = ({
-	customFields = { showImage: true, showHeadline: true, imageRatio: "3:2" },
-}) => {
-	const { isAdmin, arcSite } = useFusionContext();
-	const shouldLazyLoad = customFields?.lazyLoad && !isAdmin;
 	if (shouldLazyLoad && isServerSide()) {
 		return null;
 	}
+
+	const linkURL = content?.websites?.[arcSite]?.website_url;
+	const imageURL =
```

Comment:
We should use the util function from Arc Themes Components to get the image `getImageFromANS`

Reply:
Changed to `const imageURL = getImageFromANS(content);`